src/components/MarkDown/index.tsx

```diff
@@ -61,11 +61,7 @@ const RootRendererComponent: FunctionComponent<{
   inline?: boolean
   parentProps?: XStyledProps
 }> = ({ inline = false, parentProps = {}, children }) => (
-  <StyledContainer
-    as={inline ? 'span' : 'div'}
-    inline={inline}
-    {...parentProps}
-  >
+  <StyledContainer as="div" inline={inline} {...parentProps}>
```

Comment:
So this will always be a div now ? 

This is going to break a lot of Markdown components :)

Reply:
it should not

If inline prop is on, we css p tags to be inlined. It should be enough. We need to check if we have some places with multiple markdowns following each other but that's it?

In any case this is required as p tag inside a span tag isn't correct :(

Reply:
👌 

Btw the as="div" is useless, Box is by default a div

Reply:
Good point I fix it